db/migrate/20210415015550_create_terms_of_service_files.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateTermsOfServiceFiles < ActiveRecord::Migration[5.0]
+  def change
+    # rubocop:disable Style/SymbolProc
+    create_table :terms_of_service_files do |t|
+      t.timestamps
+    end
+    # rubocop:enable Style/SymbolProc
```

Comment:
This applies only to new tables without any data, except timestamps. Pretty rare. I just did that here because Paperclip assumes an existing table to add attachment fields to.